code/src/main/webapp/JavaScript/script.js

```diff
@@ -187,8 +187,12 @@ function attachSignin(element, negativeUser) {
     var servlet = "";
     if (negativeUser) {
       servlet = '/get-negative-user-info';
+      params.append('calendar', true);
```

Comment:
why don't we give the negative user the same control as the positive user? 

I guess we can't really do anything for them without that control.